pkg/time/rate/rate_test.go

```diff
@@ -400,7 +400,7 @@ type wait struct {
 func runWait(t *testing.T, lim *Limiter, w wait) {
 	start := time.Now()
 	err := lim.WaitN(w.ctx, w.n)
-	delay := time.Now().Sub(start)
+	delay := time.Since(start)
```

Comment:
I think this is not on our code. I guess the `Since()` is kind of new for code that should be usable on older version of golang.